src/syrupy/serializers/amber.py

```diff
@@ -126,9 +139,13 @@ def serialize_set(
 
     @classmethod
     def serialize_dict(
-        cls, data: "SerializableData", *, depth: int = 0, visited: Set[Any] = set()
+        cls,
+        data: "SerializableData",
+        *,
+        depth: int = 0,
+        visited: Optional[Set[Any]] = None,
     ) -> str:
-        kwargs = dict(depth=depth + 1, visited=visited)
+        kwargs = {"depth": depth + 1, "visited": visited}
```

Comment:
Why the change?

Reply:
`{}` is preferred over `dict()`. It's slightly faster. Also more lint rules ensures we're consistent (two developers write same code)